draft-ietf-tls-tls13.md

```diff
@@ -3184,24 +3183,10 @@ end-entity certificate remain.
     TLS_DH_anon_WITH_AES_128_GCM_SHA256   DH_anon    AES_128_GCM  SHA256
     TLS_DH_anon_WITH_AES_256_GCM_SHA384   DH_anon    AES_128_GCM  SHA384
 
-                    Key      Implicit IV   Explicit IV
-    Cipher         Material  Size          Size
-    ------------   --------  ----------    -----------
-    NULL               0          0             0
-    AES_128_GCM       16          4             8
-    AES_256_GCM       32          4             8
 
 Key Material
 : The number of bytes from the key_block that are used for
   generating the write keys.
```

Comment:
This definition can be cut too. It's just for the table you're cutting above. The only other place in the document that says "key material" is in Key Calculation, and they're not really referring to the same thing.